src/ilmm.jl

```diff
@@ -1,19 +1,18 @@
 """
-    ILMM(fs, H)
+    ILMM(f, H)
+
 An Instantaneous Linear Mixing Model (ILMM) -- a distribution over vector-
 valued functions. Let `p` be the number of observed outputs, and `m` the number of latent
 processes, then `H`, also known as the mixing matrix, is a `p x m` matrix whose column
-space spans the output space. The latent processes are represented a Gaussian process `f`.
+space spans the output space. Given a sample from `f`, the value of the ILMM at any input `x` for all outputs is
+```julia
+H * [f((x, 1)), f((x, 2)), ..., f((x, m))]
 
```

Comment:
It won't let me make this a suggestion given the suggestion uses the "```" but I think we need to close the julia code block!

Reply:
Oh, weird. I'll have to modify locally -- the suggestion API doesn't seem to like it.